db/migrate/20160829175601_add_content_filter_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddContentFilterToUser < ActiveRecord::Migration
+  def change
+    add_column :users, :content_filter, :string
```

Comment:
Good point about field length. But will leave this alone for now (unless you want me to deal with it now).